src/TropicalGeometry/matrix.jl

```diff
@@ -41,3 +41,35 @@ end
 function det(A::Matrix{TropicalSemiringElem{minOrMax}}) where {minOrMax<:Union{typeof(min),typeof(max)}}
     return det(matrix(TropicalSemiring{minOrMax},A))
 end
+
+@doc raw"""
+    tropically_generic(A::Generic.MatSpaceElem{<: TropicalSemiringElem}, minOrMax)
+Checks if a collection of vectors in the tropical torus (given as columns of a matrix `A`) are in tropical general position with respect to the `minOrMax` convention.
+# Examples
+```jldoctest
+julia> A = matrix(tropical_semiring(),[1 0;0 1])
+[(1)   (0)]
+[(0)   (1)]
+
+julia> tropically_generic(A,min)
```

Comment:
Based on this usage example I would expect this function to be called `is_tropically_generic`